setup.py

```diff
@@ -38,6 +38,10 @@
         "setuptools",
         "docutils",
         "pygments",
+        "requests"
+    ],
+    tests_require=[
+        'mock'
```

Comment:
As mentioned above, is there a reason you're not using the built-in `unittests.mock`?